Sources/src/PBISequence_source.cc

```diff
@@ -202,9 +250,26 @@ namespace mu2e {
     outE = pm_.makeEventPrincipal(runNumber, subRunNumber, eventNumber, ts, false);
 
   } // managePrincipals()
-    //----------------------------------------------------------------
+
+
+  //----------------------------------------------------------------
+  void PBISequenceDetail::computeRunDataProducts( std::vector<std::string> const& inputFiles ){
+    for ( auto const& file : inputFiles){
+      std::ifstream in(file,std::ifstream::in);
+      float protons;
```

Comment:
And "float" in the code looks ugly to me.  Not using doubles have to be justified by resource constraints, does not seem to be the case here.